pkg/platform/provider/imported/cluster/provider.go

```diff
@@ -110,6 +111,28 @@ func (p *Provider) ValidateCredential(cluster clusterprovider.InternalCluster) (
 	return allErrs, nil
 }
 
+func dialBySimpleHttp(host string, port int32, timeout time.Duration) (bool, error){
+	client := &http.Client{
+		Timeout: timeout,
+	}
+
+	url := "http://"+host+":"+strconv.Itoa(int(port))
```

Comment:
The kube-apiserver has the https service enabled by default. Using the http protocol to detect will cause a tls handshake error to the normal apiserver.